BankWallet/BankWallet/Modules/Pin/PinViewController.swift

```diff
@@ -174,9 +174,11 @@ extension PinViewController: IPinView {
 struct PinPage {
     var description: String?
     var error: String?
+    var title: Bool
```

Comment:
nit: should you rename it so more describing, as this is a bool

Reply:
> nit: should you rename it so more describing, as this is a bool

nice catch ;) @ant013 shame on you!

Reply:
@esengulov updated specially for you